python/lsst/ts/observatory/control/base_tcs.py

```diff
@@ -1105,6 +1105,30 @@ async def offset_rot(self, rot: float) -> None:
             )
         )
 
+    async def offset_pa(self, angle: float, radius: float) -> None:
+        """Offset the telescope based on a position angle and radius to the
+        current target position.
+
+        This is designed to counter rotation offset when the desired target
+        is not at the center of rotation.
```

Comment:
Does this method automatically take into account the difference between the boresight and the detector center? If not, maybe add consider adding warning here until we understand exactly what it does?